app/models/concerns/authorizable.rb

```diff
@@ -16,7 +16,7 @@ module Authorizable
       if user.nil?
         self.where('1=0')
       elsif user.admin?
-        self.scoped
+        self.all
```

Comment:
Is this equivalent?  IIRC .all will immediately query, while scoped just generates a scope.  `where(nil)` might be more equivalent.